cloud_platform_ovh/models/cloud_platform.py

```diff
@@ -1,24 +1,27 @@
 # Copyright 2017-2019 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
-import logging
+import re
+import os
 
 from odoo import api, models
 
-_logger = logging.getLogger(__name__)
+from odoo.addons.cloud_platform.models.cloud_platform import FilestoreKind
+from odoo.addons.cloud_platform.models.cloud_platform import PlatformConfig
 
-try:
-    from odoo.addons.cloud_platform.models.cloud_platform import FilestoreKind
-    from odoo.addons.cloud_platform.models.cloud_platform import PlatformConfig
-except ImportError:
-    FilestoreKind = None
-    PlatformConfig = None
-    _logger.debug("Cannot 'import from cloud_platform'")
+
+SWIFT_STORE_KIND = FilestoreKind('s3', 'remote')
```

Comment:
I have issues making the abstraction version on OVH and I think this is because this line should read
```suggestion
SWIFT_STORE_KIND = FilestoreKind('swift', 'remote')
```